package.json

```diff
@@ -19,44 +19,44 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^14.1.1",
-    "@yume-chan/adb": "^0.0.15",
+    "@types/dom-webcodecs": "^0.1.4",
+    "@types/node": "^18.8.4",
+    "@types/offscreencanvas": "^2019.7.0",
+    "@types/wicg-file-system-access": "^2020.9.5",
     "@yume-chan/adb-backend-webusb": "^0.0.15",
     "@yume-chan/adb-credential-web": "^0.0.15",
+    "@yume-chan/adb": "^0.0.15",
     "buffer": "^6.0.3",
     "i18next": "^21.8.10",
     "pre-commit": "^1.2.2",
-    "react": "^18.0.0",
     "react-cookie": "^4.1.1",
     "react-dom": "^18.0.0",
     "react-ga4": "^1.4.1",
     "react-i18next": "^11.17.2",
     "react-redux": "^8.0.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "react": "^18.0.0",
+    "stream-data-view": "^1.7.2",
+    "typescript": "<4.7.0",
     "uuid": "^8.3.2",
     "web-vitals": "^2.1.4",
     "with-staged": "^1.0.2",
     "xterm-addon-fit": "^0.5.0",
     "xterm-for-react": "^1.0.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "ESLINT_NO_DEV_ERRORS=true react-scripts start",
```

Comment:
Getting bounced away with lint errors while building stuff was super annoying, but I think it's worth suggesting it as the default? Lint runs on commit anyway! 😄

Reply:
Yeah, that is totally fine with me.